Controllers/comments.controllers.js

```diff
@@ -0,0 +1,19 @@
+const { deleteComment } = require("../models/comments.models");
+
+exports.removeComment = (req, res, next) => {
+  const id = req.params.comment_id;
+  deleteComment(id)
+    .then(() => {
+      res.status(204).send();
+    })
+    .catch((err) => {
+      if (err.code === "22P02") {
```

Comment:
A note on your error handling across the project:

There are a fair amount of catches in your controllers that do similar error handling with error codes and it is quite 'wet' at times. We want it to be as DRY as possible which is why handling errors in the middleware can be super attractive. This is mainly to do with repeated error codes like '22P02'.

What would be ideal is that you just let those '22P02' codes filter through your catch and straight to an error handling middleware that specifically sends an identical message so you improve the conciseness of the controller and the dryness of the code.

**NB:** I realise you have designed these catches to send specific messages for the context the error was thrown in. So I can imagine you won't refactor, that's fine. Hopefully you haven't been told this multiple times already 😂 In an ideal world all your invalid ID messages would be more generic like 'invalid ID provided, please provide number' and that badly formed votes body would send a 'bad request' message